WordPress/gutenberg
# Font Library: Improve 'No fonts installed' state when fonts are installed but not activated

## What?

Fixes https://github.com/WordPress/gutenberg/issues/63418

## Why?

Styles panel shows a 'No fonts installed.` text event when fonts are installed, but not activated. 

## How?

Added condition to display message conditionally. Now it will display  'No fonts activated.` when fonts are installed but not activated.

## Testing Instructions

- Open Styles panel , go to typography
- If you don't have any font installed, it should show  'No fonts installed.`
- If you have any font installed, it should show  'No fonts activated.`
